app/src/main/AndroidManifest.xml

```diff
@@ -38,6 +38,16 @@
                 android:value="" />
         </activity>
 
+        <activity
+            android:name=".ui.screens.newsbrainz.NewsBrainzActivity"
+            android:exported="false"
```

Comment:
We don't actually need to specify this until and unless an intent filter is added to the activity.